alignak/external_command.py

```diff
@@ -2973,49 +2973,61 @@ def process_host_check_result(self, host, status_code, plugin_output):
         :return: None
         TODO: say that check is PASSIVE
         """
-        # raise a PASSIVE check only if needed
+        now = time.time()
+        cls = host.__class__
+
+        # If globally disabled OR host disabled, do not launch..
+        if not cls.accept_passive_checks or not host.passive_checks_enabled:
+            return
+
+        try:
+            plugin_output = plugin_output.decode('utf8', 'ignore')
+            logger.debug('%s > Passive host check plugin output: %s',
+                         host.get_full_name(), plugin_output)
+        except UnicodeError:
+            pass
```

Comment:
It can be nice to have a warning / error

Reply:
It is of no interest... except for spamming the log 😉